examples/slice/main.go

```diff
@@ -0,0 +1,17 @@
+package main
+
+import (
+	"fmt"
+	"github.com/bitfield/script"
+	"log"
+)
+
+func main() {
+	// Get a Slice with all the running process PID
+	pids, err := script.Exec("ps -ea").Column(1).Slice()
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	fmt.Printf("Running process PIDs are:  %q", pids)
```

Comment:
Okay, but this doesn't really make use of them as a _slice_, does it? If we want to demonstrate how useful it is to turn a pipe into a slice, we could `range` over that slice, for example.